### DISCUSSION TITLE
“Send” in PR review comment UI confusing

### DISCUSSION THREAD

**Discussion by toshok** [community/community#3054] [Mobile]
![23BD4807-002A-4406-B58D-28511179645C](https://user-images.githubusercontent.com/24245/102957714-fbc5fa00-448f-11eb-84d3-a9218e0029bb.jpeg)

this would/will be 100% answered just by clicking “Send”, but it’s non-obvious what the behavior here will be: will it add a single comment to the PR, or will it start a review?

**Comment by rnystrom** [comment]
Great feedback. It will only start a review, for simplicity's sake we do not support "Single comment"-style reviews on the mobile app. Would it be more clear if the button said "Start review" or "Add" if a review was already started?
